packages/office-ui-fabric-react/etc/office-ui-fabric-react.api.md

```diff
@@ -494,7 +494,7 @@ export class Calendar extends BaseComponent<ICalendarProps, ICalendarState> impl
 }
 
 // Warning: (ae-forgotten-export) The symbol "ICalloutState" needs to be exported by the entry point index.d.ts
-//
+// 

```

Comment:
This seems unintended?

Reply:
Auto generated file. Didn't add manually.